src/pages/news.js

```diff
@@ -6,20 +6,37 @@ import { NewsSummary } from "../components/News";
 import Link from "../components/Link";
 import Seo from "../components/seo";
 
-const NewsCard = ({ frontmatter, fields }) => {
-  const { title, external, categories, description, thumbnail, avatar, author, date } = frontmatter;
-  const link = external ? { href: external } : { to: fields.slug };
+const NewsCard = (props) => {
+  const {
+    title,
+    published_at,
+    authors,
+    canonical_url,
+    slug,
+    feature_image_local,
+    tags,
+    custom_excerpt,
+    meta_description,
+    excerpt,
+  } = props;
+
+  const { name, profile_image_local } = authors[0];
+  const link = canonical_url ? { href: canonical_url } : { to: slug };
+
+  const description = custom_excerpt || meta_description || excerpt;
+
+  // TODO: limit description to x many characters.
```

Comment:
Might be better to limit to X lines with something like this:
```css
.description {
    display: -webkit-box;
    -webkit-line-clamp: 4; /* widely available despite the prefix */
    -webkit-box-orient: vertical;
    overflow: hidden;
}
```
* [before](https://user-images.githubusercontent.com/3853033/159893113-91bbf8d3-2aeb-454c-830d-5641cea59b37.png) vs [after](https://user-images.githubusercontent.com/3853033/159893299-56bd65f1-0336-4ed7-9379-d1b38fac7233.png)
* [reference](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp)

I can add it, but didn't want to mess with the PR at this point, since Netlify is picking up new commits (not sure if it would get replaced, or a new instance would spin up; got no experience with Netlify).

Let me know if there's a green light to do so :)

Reply:
Yeah, let's add this.